etl/example.rkt

```diff
@@ -0,0 +1,16 @@
+#lang racket
+
+(provide (contract-out
+          [etl (-> hash? hash?)]))          
+
+; TODO Provide contract on the keys and values of the input and output
+
+(define (etl h)
```

Comment:
Looks good, but you can use `for*/hash` to build the result and `in-hash` to walk the old table faster.

```
(define (etl h)
    (for*/hash ([(score letter*) (in-hash h)]
                [letter (in-list letter*)])
```